### PR TITLE
style and behavior refinements for new settings

### PR DESCRIPTION
**Pull request by johngruen**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: f22536c8210d82557ee5a59a26b782b7eb9fee39
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: app/scripts/views/settings.js [hunk 1]
```diff
@@ -81,9 +83,10 @@ function ($, modal, Cocktail, p, Session, FormView, BaseView, AvatarMixin,
 
     context: function () {
       var account = this.getSignedInAccount();
+      console.log(account);
```
Comment (vladikoff) [inline app/scripts/views/settings.js]: console.log needs to be removed.

Comment (johngruen) [inline app/scripts/views/settings.js]: woops

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: fbf2aee7c734f73cd59c34b7092dff1187fae524
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/mozilla/fxa-content-server/blob/master/CONTRIBUTING.md#git-commit-guidelines

---
This message was auto-generated by https://gitcop.com

File: app/scripts/views/settings.js [hunk 2]
```diff
@@ -127,7 +129,7 @@ function ($, modal, Cocktail, p, Session, FormView, BaseView, AvatarMixin,
     _onProfileChange: function () {
       var account = this.getSignedInAccount();
       this._showAvatar();
-      this.$('.username').text(account.get('displayName') || account.get('email'));
+      this.$('.username').text(account.get('email'));
```
Comment (zaach) [inline app/scripts/views/settings.js]: We can remove this since we're no longer showing the display name.

**Pull request merged**